#! /bin/sh /usr/share/dpatch/dpatch-run
## build-filename.dpatch
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: https://sourceforge.jp/tracker/index.php?func=detail&aid=12568&group_id=1720&atid=6435
## DP: Missing terminating NULL for g_build_filename call.

@DPATCH@
diff -urNad gsynaptics-0.9.14~/src/main.c gsynaptics-0.9.14/src/main.c
--- gsynaptics-0.9.14~/src/main.c	2008-02-28 23:27:28.000000000 +0100
+++ gsynaptics-0.9.14/src/main.c	2008-05-17 08:34:04.000000000 +0200
@@ -690,7 +690,7 @@
 
 	gtk_window_set_default_size (GTK_WINDOW (widget), 400, -1);
 
-	path = g_build_filename (DATADIR "/pixmaps", "touchpad.png");
+	path = g_build_filename (DATADIR, "pixmaps", "touchpad.png", NULL);
 	if (path)
 	{
 		icon_pixbuf = gdk_pixbuf_new_from_file (path, NULL);
